### ISSUE TITLE
Blurry Product Images

### ISSUE THREAD

**Issue by SeanIsAtWork** [ancaipen/Provia#13]
With the use of products across the website, "thumbnails" from the product are used when displayed on a page. Currently these images are blurry at a seemingly random rate. This was something Natalee resolved by changing the aspect ratio of the product thumbnails from original to 3:2 which eliminates blurriness but we would like the full aspect ratio of any given product image to be utilized to avoid unwanted cropping of photos.

![image](https://user-images.githubusercontent.com/95431522/145281263-cc852df7-9c84-45cd-9594-35ec3b40cd40.png)

**Comment by ncaipen** [comment]
I can't find the exact images that you have displayed above, but when I go to the siding page and look at the Dutch siding there was a blurry image that's different than the one above.  When you look at the image dimensions it is displaying much smaller than it should, so I think that's why the image is stretched and blurry.  I resaved the image and uploaded again to the product and it's displaying the larger size now and is no longer blurry.  I would check your image dimensions for any that are blurry and then upload again with a new name if it's displaying small when inspecting on the live site.  If min image sizes were changed after these images were uploaded that may be why some are fine and others are stretched.
